boards/arm/stm32l4/stm32l476vg-disco/src/stm32_appinit.c

```diff
@@ -269,6 +269,7 @@ int board_app_initialize(uintptr_t arg)
       snprintf(blockdev, 18, "/dev/mtdblock%d", N25QXXX_MTD_MINOR);
       snprintf(chardev, 12, "/dev/mtd%d", N25QXXX_MTD_MINOR);
 
+#if defined(CONFIG_BCH)
```

Comment:
should we move this statement before line 269 too? Since blockdev and chardev isn't really used too in case CONFIG_BCH = n. The guard at line 137 need to change too.

Reply:
good point. fixed.